_docs-sources/foundations/accounts/prerequisites.md

```diff
@@ -0,0 +1,70 @@
+# Prerequisites
+
+In order to enable control tower you will need the following:
+
+1. A new AWS Account and a user with administrator permissions. (We recommend using an IAM user with admin permissions rather than the root user)
+
+  :::info
+  This account will become the root of your multi-account setup after enabling Control Tower.
+  :::
+
+1. Three(3) new unique email addresses for your logs, shared, and security(audit) accounts.
+1. A KMS Key for encrypting Control Tower Resources with a suitable permissions policy (See Guide Below)
+
+## KMS Key Creation
+:::info
+For more help setting up KMS see the AWS docs: [Guidance for KMS keys](https://docs.aws.amazon.com/en_us/controltower/latest/userguide//kms-guidance.html)
+:::
+1. Navigate to KMS in your root AWS account using a user with permission to create KMS Keys
+1. Ensure you are in your home region and click `Create Key`
```

Comment:
Should we explain what the home region is in the top section? We only spell it out in the `enable-control-tower` doc